chain/genesis.json

```diff
@@ -1,6 +1,6 @@
 {
   "genesis_time": "2023-04-10T18:42:34Z",
-  "chain_id": "elystestnet-1",
+  "chain_id": "elysicstestnet-1",
```

Comment:
I think we should remove this file. The read me says it will be moved to appropriate directory which is `networks`

Reply:
yep make sense removed